cmd/gateway-common.go

```diff
@@ -432,7 +432,7 @@ type MetricsTransport struct {
 // RoundTrip implements the RoundTrip method for MetricsTransport
 func (m MetricsTransport) RoundTrip(r *http.Request) (*http.Response, error) {
 	metered := shouldMeterRequest(r)
-	if metered && (r.Method == http.MethodGet || r.Method == http.MethodHead) {
+	if metered && (r.Method == http.MethodPost || r.Method == http.MethodPut) {
```

Comment:
shouldn't we do this for all calls? @poornas - the idea was to differentiate how many `gets/heads` were made to backend - so we need to add other metrics to capture the data sent?

Reply:
If the intent is to just capture the number of gets/heads made to backend gateway_x_bytes_sent metric is not very useful as it will always be zero for get/head and not intuitively named

Reply:
We should make it clearer and break it away from existing implementation if it doesn't make sense.